blog/templates/blog/base.html

```diff
@@ -9,15 +9,20 @@
     </head>
     <body>
         <header class="page-header">
-          <div class="container">
-              <h1><a href="/">Django Girls Blog</a></h1>
-          </div>
+            <div class="container">
```

Comment:
Why did you change the indentation here?

Reply:
Edited

Reply:
What did you edit exactly and how?

Reply:
I'm sorry I didn't understand what I should correct